build/yamls/antrea.yml

```diff
@@ -2416,7 +2416,9 @@ data:
     featureGates:
     # Enable AntreaProxy which provides ServiceLB for in-cluster Services in antrea-agent.
     # It should be enabled on Windows, otherwise NetworkPolicy will not take effect on
-    # Service traffic.
+    # Service traffic. Note that, due to the implementation of Service in AntreaProxy, the maximum
+    # Endpoints that AntreaProxy can support now is 800. If the Endpoints of Service exceed 800, the
+    # exceeding Endpoints will be ignored.
```

Comment:
Thinking more about this, we should document this in feature-gates.md IMO, not necessarily in the yaml manifest. @jianjuns @tnqn?

Reply:
Sorry, I forgot that @tnqn suggested me that this should be documented in feature-gates.md.